tensorflow/compiler/xla/service/llvm_ir/ir_builder_mixin.h

```diff
@@ -71,7 +71,7 @@ class IrBuilderMixin {
 
   template <class... Args>
   llvm::Value* BitCast(Args&&... args) {
-    return mixin_builder()->CreaterBitCast(std::forward<Args>(args)...);
+    return mixin_builder()->CreateBitCast(std::forward<Args>(args)...);
```

Comment:
it's better to reuse the previous commit and update the history via `git push --force`.